google-cloud-spanner/pom.xml

```diff
@@ -69,6 +69,7 @@
           <systemPropertyVariables>
             <spanner.testenv.config.class>com.google.cloud.spanner.GceTestEnvConfig</spanner.testenv.config.class>
             <spanner.testenv.instance>projects/gcloud-devel/instances/spanner-testing</spanner.testenv.instance>
+            <spanner.gce.config.project_id>gcloud-devel</spanner.gce.config.project_id>
```

Comment:
Won't this break running the integration tests locally if you don't have permission to access the `gcloud-devel` project, unless you specify a different value for this system property in your maven command? Currently the integration tests use the Google Cloud project that is specified in your environment in combination with the credentials that have also been configured in your environment.

Reply:
I thought so, but the test instance is already hardcoded to `projects/gcloud-devel/instances/spanner-testing` (see property `spanner.testenv.instance`). I think that the only way you can run the IT tests currently is by overriding the two properties. Maybe we should update the CONTRIBUTING.md file here as well.

Reply:
Yep, you're right. My local config overrides the `spanner.testenv.instance` property. This change would mean that I would also need to override the `spanner.gce.config.project_id` property, but that is not really a big ask when the first one needs to be overridden anyways. As you noted, this might be worth noting in the CONTRIBUTING.md.

Reply:
Thanks, updated the `CONTRIBUTING.md` with the new integration tests instructions.